src/main/java/com/google/api/codegen/util/java/JavaTypeTable.java

```diff
@@ -14,6 +14,8 @@
  */
 package com.google.api.codegen.util.java;
 
+import static com.google.api.codegen.util.java.JavaTypeTable.JavaLangResolution.ESCAPE_JAVA_LANG_CLASH;
```

Comment:
It seems like `JavaLangResolution` should actually be here, since this is the class that uses it for logic.